.pre-commit-config.yaml

```diff
@@ -25,6 +25,7 @@ repos:
     hooks:
       - id: ruff
         exclude: "tests"
+        args: ["--fix"]
```

Comment:
Would this still fail the PR if the files are not formatted correctly? Current use run `pre-commit` within tox. And, the tox environment is used in `pre_merge` checks.